doc/user/data/sql_funcs.yml

```diff
@@ -278,6 +278,12 @@
     description: Converts Unix epoch (seconds since 00:00:00 UTC on January 1, 1970)
       to timestamp
 
+  - signature: 'to_char(val: timestamp, format: str)'
+    description: >-
+      Converts a timestamp into a string using the specified format, see the
+      detailed docs for the full specification language.
```

Comment:
I think we can drop the "see the detailed docs for the full specification language" clause for consistency with the other descriptions that have a more detailed URL. (Also it's a run-on sentence.)